e2e-tests/src/cells/list.rs

```diff
@@ -0,0 +1,88 @@
+#[cfg(test)]
+mod test {
+    use client::Client as AuraeClient;
+    use proto::cells::{Cell, CellGraphNode, CellServiceListResponse};
+
+    use crate::common::{
+        helpers::{allocate_cell, list_cells},
+        request_builders::CellServiceAllocateRequestBuilder,
+    };
+
+    #[tokio::test]
+    #[ignore = "this test requires a fresh Aurae daemon which right now we cannot guarantee"]
```

Comment:
Yes, we can probably use some of the  code from the other PR to deal with this. dma brought up some concerns though in the other PR, and I haven't formulated a response yet.

Re a "fresh" auraed, that isn't hard to get, but we will need to keep in mind whether the system is fresh or not and if that effects what we are testing. Cells should have a fresh system (best effort to clean up on `drop`), but I haven't looked into the other services yet.